README.md

```diff
@@ -1 +1,139 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Rails-blog-app<a name="about-project"></a>
+
+**Rails-blog-app** 
+The Rails-blog-app will be a classic example of a blog website. It will be a functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby on Rails</summary>
+<ul>
+  <li>Ruby</li>
+  <li>Ruby</li>
+</ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Ruby on Rails**
+- **Functional website**
+- **Unit testing**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+> You need the following tools be installed in your computer:
+>
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+> - [Rails](https://www.tutorialspoint.com/ruby-on-rails/rails-installation.htm)
+
+
+### Setup
+1. Download the `Zip` file or **Clone the repo with**:
+
+```bash
+git clone https://github.com/hafiz1379/Rails-blog-app.git
+```
+
+2. To access the project directory
+
+```bash
+cd rails-blog-app
+```
+### Install
+
+```bash
+bundle install
+rails db:migrate
+```
+
+### Run the seeder
+
+```
+rails db:seed
+```
+
+### Run the App
+
+```bash
+rails s
+```
+
+```
+Check http://localhost:3000/ in your browser
+```
+### Run Test
+
+- To run models tests
+```bash
+rspec
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Alejandro**
+- GitHub: [Alejandro](https://https://github.com/hafiz1379)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hafiz1379)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Future Features <a name="key-features"></a>
+
+[ ] Add functionality.
+[ ] Preserve data.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+### Contributions, issues, and feature requests are welcome!
+
+Feel free to check [issues page](https://github.com/Alejandro-Bernal-M/rails-blog-app/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for teaching me and my lovely girlfriend and family for supporting me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job documenting your project 👏 👏 👏 👏  , However a professional README documentation should not have broken link 😄 , Kindly fix the broken link attached to the MIT license 👍 

![Capture](https://github.com/hafiz1379/Rails-blog-app/assets/80458199/9cad6e4f-2443-4c01-b17b-90f3cf0555e9)